src/diffpy/labpdfproc/labpdfprocapp.py

```diff
@@ -152,7 +152,11 @@ def _define_arguments():
 def _add_mud_selection_group(p, is_gui=False):
     """Current Options:
     1. Manually enter muD (`--mud`).
-    2. Estimate muD from a z-scan file (`-z` or `--z-scan-file`).
+    2. Estimate from a z-scan file (`-z` or `--z-scan-file`).
+    3. Estimate theoretically based on sample mass density
+    (`-td` or `--theoretical-from-density`).
```

Comment:
check but this is breaking the usual pattern that it should be `-l` (one letter) or `--letters` (multiple letters need multiple dashes.